droolsjbpm-integration-tests/pom.xml

```diff
@@ -0,0 +1,43 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.drools</groupId>
+    <artifactId>droolsjbpm-integration</artifactId>
+    <version>7.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>droolsjbpm-integration-tests</artifactId>
+  
+  <dependencies>
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-compiler</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.jbpm</groupId>
+        <artifactId>jbpm-bpmn2</artifactId>
+        <scope>test</scope>
+    </dependency>
+    <dependency><!-- For unit test logging: configure in src/test/resources/logback-test.xml -->
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+  <build>
+    <plugins>
+      <!-- This is a module for JUnit unit test only, no need to generate a JAR. -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-jar</id>
+            <phase>none</phase>
+          </execution>
+        </executions>
```

Comment:
@psiroky as discussed on IRC, summary:

- the `-sources` is not generated. because jar generation itself is skipped already
- the `-tests` and `-test-sources` indeed I want it generated, for two reasons 1 convenience for user instead of checking out the github, 2 especially later to reuse code
- the `-javadoc` I tested it (below) and indeed not generated because jar generation itself is skipped already

```
[mmortari@mmortari droolsjbpm-integration]$ mvn -Dfull install -pl droolsjbpm-integration-tests -am
...
[INFO] --- maven-javadoc-plugin:2.10.3:jar (default) @ droolsjbpm-integration-tests ---
[INFO] 
[INFO] --- maven-failsafe-plugin:2.18.1:integration-test (default) @ droolsjbpm-integration-tests ---
[INFO] 
[INFO] --- maven-failsafe-plugin:2.18.1:verify (default) @ droolsjbpm-integration-tests ---
[INFO] Failsafe report directory: /home/mmortari/git/droolsjbpm-integration/droolsjbpm-integration-tests/target/failsafe-reports
[INFO] 
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ droolsjbpm-integration-tests ---
[INFO] No primary artifact to install, installing attached artifacts instead.
[INFO] Installing /home/mmortari/git/droolsjbpm-integration/droolsjbpm-integration-tests/pom.xml to /home/mmortari/.m2/repository/org/drools/droolsjbpm-integration-tests/7.0.0-SNAPSHOT/droolsjbpm-integration-tests-7.0.0-SNAPSHOT.pom
[INFO] Installing /home/mmortari/git/droolsjbpm-integration/droolsjbpm-integration-tests/target/droolsjbpm-integration-tests-7.0.0-SNAPSHOT-tests.jar to /home/mmortari/.m2/repository/org/drools/droolsjbpm-integration-tests/7.0.0-SNAPSHOT/droolsjbpm-integration-tests-7.0.0-SNAPSHOT-tests.jar
[INFO] Installing /home/mmortari/git/droolsjbpm-integration/droolsjbpm-integration-tests/target/droolsjbpm-integration-tests-7.0.0-SNAPSHOT-test-sources.jar to /home/mmortari/.m2/repository/org/drools/droolsjbpm-integration-tests/7.0.0-SNAPSHOT/droolsjbpm-integration-tests-7.0.0-SNAPSHOT-test-sources.jar
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Drools and jBPM integration multiproject ........... SUCCESS [  2.387 s]
[INFO] droolsjbpm-integration-tests ....................... SUCCESS [  2.488 s]
[INFO] ------------------------------------------------------------------------
```
(please note I've included the javadoc plugin part in the log, showing no javadoc are generated, no jar = no javadocs.)